lib/rubocop/cop/style/disjunctive_assignment_in_constructor.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks constructors for disjunctive assignments that should
+      # be plain assignments.
+      #
+      # So far, this cop is only concerned with disjunctive assignment of
+      # instance variables.
+      #
+      # In ruby, an instance variable is nil until a value is assigned, so the
+      # disjunction is unnecessary. A plain assignment has the same effect.
+      #
+      # @example
+      #   # bad
+      #   def initialize
+      #     @x ||= 1
+      #   end
+      #
+      #   # good
+      #   def initialize
+      #     @x = 1
+      #   end
+      class DisjunctiveAssignmentInConstructor < Cop
+        include DefNode
+
+        MSG = 'Unnecessary disjunctive assignment. Use plain assignment.'.freeze
+
+        def on_def(node)
+          check(node)
+        end
+        alias on_defs on_def
```

Comment:
I think that aliasing to `defs` seems unnecessary because `initialize` method is an instance method.